### PR TITLE
costume-ball: add exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**
@kytrinyx This is a new exercise. I've made a marginal effort to come up with a backstory. I don't know if you think this is headed in the right direction.

I'm also not well versed in licensing. This is the source that I used: https://github.com/mmalita/PrologPuzzles The puzzles can also be found here: https://ypologist.com/mmalita17/HOMEPAGE/logic/index.html

### CODE REVIEW

**Comment by kytrinyx** [comment]
I think we'd need to ask for permission to use these, since they're not released under a license that would allow sharing (as far as I can tell). Maybe we could email Jan Wielemaker as listed in the README?

**Comment by ErikSchierboom** [comment]
> Maybe we could email Jan Wielemaker as listed in the README?

Done! I could even email him in Dutch :)

File: exercises/practice/costume-ball/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "test": [
+      "costume_ball_tests.plt"
+    ],
+    "example": [
+      ".meta/costume_ball.example.pl"
+    ]
+  },
+  "blurb": "Solve the Costume Ball puzzle.",
+  "source": "PrologPuzzles",
+  "source_url": "https://github.com/mmalita/PrologPuzzles"
```
Comment (ErikSchierboom) [inline exercises/practice/costume-ball/.meta/config.json:18]:
```suggestion
  "source": " Dell - Math Puzzles Logic Problems, Jan 2002 p.15"
```
```

**Review by kytrinyx — APPROVED** [review]
If the copyright holder says we can use this, then let's go ahead!

**Pull request closed without merging**